src/fiatExchanges/CoinbasePaymentSection.tsx

```diff
@@ -13,28 +13,43 @@ import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import colors from 'src/styles/colors'
 import fontStyles from 'src/styles/fonts'
+import { useTokenInfo } from 'src/tokens/hooks'
 import { walletAddressSelector } from 'src/web3/selectors'
+import { getNetworkFromNetworkId } from 'src/web3/utils'
 
 export interface CoinbasePaymentSectionProps {
   cryptoAmount: number
   coinbaseProvider: FetchProvidersOutput
   appId: string
   analyticsData: ProviderSelectionAnalyticsData
+  tokenId: string
 }
 
 export function CoinbasePaymentSection({
   cryptoAmount,
   coinbaseProvider,
   appId,
   analyticsData,
+  tokenId,
 }: CoinbasePaymentSectionProps) {
   const { t } = useTranslation()
   const walletAddress = useSelector(walletAddressSelector)!
+  const tokenInfo = useTokenInfo(tokenId)
 
+  if (!tokenInfo) {
+    // should never happen
+    return null
+  }
   // Using 'CGLD' as temp replacement for CiCoCurrency.CELO – digitalAsset
+
+  const network = getNetworkFromNetworkId(tokenInfo.networkId)!
```

Comment:
can we use a static map instead and avoid the null assertion operator? We also have a static map of network id to network, but I think it's safest just define a specific map here.
https://github.com/valora-inc/wallet/blob/5af717e768cef266ee09c190726cd626e48c20fb/src/web3/networkConfig.ts#L386-L391